base/server/examples/banner/banner.txt

```diff
@@ -0,0 +1,5 @@
+WARNING!
+Access to this service is restricted to those individuals with
+specific permissions. If you are not an authorized user, disconnect
+now. Any attempts to gain unauthorized access will be prosecuted to
+the fullest extent of the law.
```

Comment:
A bit ominous ;-).  How about the following instead?
"Access to this service is restricted to those individuals with specific permissions.  Any attempts to gain access, authorized or not, and all operations that follow, are audited and could be used against you in the court of law."

Reply:
Sure, I updated the PR and these pages too:
https://github.com/dogtagpki/pki/wiki/Access-Banner
https://github.com/dogtagpki/pki/wiki/PKI-10.4-Access-Banner-Design
https://github.com/dogtagpki/pki/wiki/PKI-Server-Banner-CLI
https://github.com/dogtagpki/pki/wiki/PKI-CLI-Options
https://github.com/dogtagpki/pki/wiki/PKI-Console